core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -551,5 +551,25 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
       )
   }
 
+  mockServer.when(
```

Comment:
I don't like how far away this is from the test that cares about this response, but I think I mentioned that in the previous PR and you said you'd rather defer that refactoring.